platform/darwin/scripts/generate-style-code.js

```diff
@@ -403,7 +403,7 @@ global.describeValue = function (value, property, layerType) {
         case 'boolean':
             return value ? '`YES`' : '`NO`';
         case 'number':
-            return 'the float ' + formatNumber(value);
+            return 'the float ' + '`' + formatNumber(value) + '`';
```

Comment:
To be pedantic, `formatNumber()` can go away if we’re going to present the number as a chunk of code. Otherwise, theoretically, it could be something like `−1` or `1,000`, whereas the literal would need to be `1000` or (in Swift) `1_000`.